assignment09/part01/process.php

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 1</title>
+</head>
+<body>
+
+<?php
+    $weight = $_POST['weight'];
+    $height = $_POST['height'];
+
+    //formula for calculating BMI:
+    //BMI = (Weight in Pounds / (Height in inches x Height in inches)) x 703
+    //source: http://www.bmi-calculator.net/bmi-formula.php
+
+    $BMI = ($weight / ($height * $height)) * 703;
+
+    if($BMI < 10){
+        $label_value = "underweight";
+    } else if ($BMI > 30){
+        $label_value = "obese";
+    } else {

```

Comment:
I like that you test for one bound, then the other and finally know that with the else you have to be in the middle. well done. 👍